### PR TITLE
Created TC 177(a-h)

### PR DESCRIPTION
**Pull request by isenkovska**

### CODE REVIEW

File: test/extended/reset.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ describe('Reset button suit', function () {
             inputValues4(name.default, gender.she, age.default, story.comedy);
             $(sel.submit).click();
             $(sel.tryAgain).click();
-            let ageField = $(sel.name).getText();
+            let ageField = $(sel.name).getText();  //wrong selector for age field
```
Comment (isenkovska) [inline test/extended/reset.js:61]: @EAGauss selector for age field has to be fixed

Comment (EAGauss) [reply test/extended/reset.js:61]: Okay. That's your branch.
Could you please fix that yourself?

**Review by EAGauss — CHANGES_REQUESTED** [review]
If you find a bug in my code, you should fix that in your branch or tell me to do that in slack or create a task in jira and assign it to me.
We are expected to push code to the main branch and discuss bugs we found in the comments in GitHub, not inside the code.

**Review by o-akhadova — CHANGES_REQUESTED** [review]
Can you correct the wrong quotes, please? 

Screenshot_3

File: test/extended/reset.js (hunk 1 above)
Comment (o-akhadova) [reply test/extended/reset.js:61]: I fixed it in my 1552 branch

**Pull request merged**